integration-tests/pkg/mock_sensor/server.go

```diff
@@ -489,9 +489,33 @@ func (m *MockSensor) pushEndpoint(containerID string, endpoint *sensorAPI.Networ
 // translateAddress is a helper function for converting binary representations
 // of network addresses (in the signals) to usable forms for testing
 func (m *MockSensor) translateAddress(addr *sensorAPI.NetworkAddress) string {
+	address := utils.IPAddress{}
+	ipNetwork := utils.IPNetwork{}
+
+	addressData := addr.GetAddressData()
+	if len(addressData) > 0 {
+		address = utils.IPFromBytes(addressData)
+	} else {
+		// If there is no address data IpNetwork should be set and represent
+		// a CIDR block or external IP address.
+		ipNetworkData := addr.GetIpNetwork()
+		if len(ipNetworkData) > 0 {
+			ipNetwork = utils.IPNetworkFromCIDRBytes(ipNetworkData)
+			// If the prefix length is 32 this is a regular IP address
+			// and not a CIDR block
```

Comment:
So what happens if we get an IPv6 address?

Reply:
The code now hands IPv6.